include/internal_trusted_storage.h

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
I did not add these include files here as they might conflict with the include files provided by TF-M

Reply:
Will discuss this with Gerard.